src/index.js

```diff
@@ -1,4 +1,4 @@
-import css from "../dist/styles.css";
+import css from "./styles.scss?inline";
```

Comment:
The component mounts the CSS string inside a `<style>` tag. The `?inline` flag tells Vite to just load the string into this variable, rather than auto-injecting it in runtime.

Reply:
neat! is this a vanilla JS thing? or a vite thing?

Reply:
Vite thing!

https://vitejs.dev/guide/features.html#disabling-css-injection-into-the-page